FEC/client/src/components/StockChart.jsx

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import LineChart from './LineChart.jsx';
+import styled from 'styled-components';
+
+//STYLING
+const OneDTag = styled.a`
+   margin: 0px 12px;
+
+   &:hover {
+    color: #21CE99;
+   }
+
+   border-bottom: ${props => props.type === '1D' ? '#21CE99 1px solid' : 'none'};
+   color: ${props => props.type === '1D' ? '#21CE99' : 'black'};
+`;
+const OneWTag = styled.a`
+   margin: 0px 12px;
+
+   &:hover {
+    color: #21CE99;
+   }
+
+   border-bottom: ${props => props.type === '1W' ? '#21CE99 1px solid' : 'none'};
+   color: ${props => props.type === '1W' ? '#21CE99' : 'black'};
+`;
+
+const OneMTag = styled.a`
+   margin: 0px 12px;
+
+   &:hover {
+    color: #21CE99;
+   }
+
+   border-bottom: ${props => props.type === '1M' ? '#21CE99 1px solid' : 'none'};
```

Comment:
I'm sure you've already thought about this, but I see a lot of duplicate styling code. Perhaps you could reuse this cleverly so that your components receive styling from a state passed down from props. Or something similar.

Reply:
I agree, keep it DRY! the reason I have duplicate styling is bc each of these elements needs to have a specific functionality when clicked, if the type on the state is equal to the element name, it will change the styling. I was unable to have one styled component that could update each of the 6 types